README.md

```diff
@@ -20,6 +20,25 @@ ReactJS Dallas is a safe place for anyone who wants to help build and develop an
 
 The default Gatsby starter.
 
+## Development Setup
+
+1. Fork and Clone Repository: `git clone --depth=1 https://github.com/<your-username>/reactjs-dallas-site.git`
+2. Setup up repo and install dependencies: `yarn`
+3. Create a new firebase project, in the overview screen click '</>' to find API key and other variables
+4. Create a .env file with the following variables (use the variables from your firebase project):
+```
+GATSBY_FIREBASE_API_KEY=
```

Comment:
Good idea! Let's also merge #15 to be able to have another way to get the `.env` values into the hands of anyone using this project. 😄